dicss.js

```diff
@@ -72,7 +72,12 @@
 	window.DICSS = {
 		"putIn": function(selector, properties) {
 			if (properties !== void 0) {
-				addRule.apply(this, arguments);
+				var consent = confirm("Are you sure you consent to the insertion of DICSS?");
+				if (consent == true) {
+					addRule.apply(this, arguments);
+				} else {
+					alert("CONSENT VIOLATED!");
```

Comment:
While `alert` is a good way to display a yell, which might be adequate in this situation, it is highly unrecommended. A modal that shows the warning while sends a police report in the backend would be more suitable for this.